client/landing/stepper/index.tsx

```diff
@@ -18,8 +18,8 @@ import { createRoot } from 'react-dom/client';
 import { Provider } from 'react-redux';
 import { BrowserRouter, matchPath } from 'react-router-dom';
 import { requestAllBlogsAccess } from 'wpcom-proxy-request';
-import { setupErrorLogger } from 'calypso/boot/common';
 import { setupLocale } from 'calypso/boot/locale';
+import { setupErrorLogger } from 'calypso/boot/setup-error-logger';
```

Comment:
A better location for this module would be `calypso/lib/error-logger`. The `boot` folder should contain only the minimal boot code and import most stuff from elsewhere. Let's not repeat the mistake when we stuffed the entire Stepper framework into `client/landing` 🙂